test/framework/src/main/java/org/opensearch/test/OpenSearchIntegTestCase.java

```diff
@@ -1928,12 +1930,16 @@ private ExternalTestCluster buildExternalCluster(String clusterAddresses, String
             InetAddress inetAddress = InetAddress.getByName(url.getHost());
             transportAddresses[i++] = new TransportAddress(new InetSocketAddress(inetAddress, url.getPort()));
         }
+        Collection<Class<? extends Plugin>> nodePlugins = nodePlugins();
```

Comment:
Please revert that, the mock plugins has to be injected using 
```
protected Collection<Class<? extends Plugin>> getMockPlugins() {
```

Reply:
Done.